install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -345,6 +345,10 @@ cleanBpfState: false
 # WARNING: Use with care!
 cleanState: false
 
+# -- Wait for KUBE-PROXY-CANARY iptables rule to appear in "wait-for-kube-proxy"
+# init container before launching cilium-agent.
```

Comment:
Should we mention/reference the issue to give context for why someone would want this?